cre.cpp

```diff
@@ -3318,6 +3318,18 @@ static int regularizeRegisteredFontsWeights(lua_State *L) {
 	return 0;
 }
 
+static int checkRegex(lua_State *L) {
+	const char *l_pattern   = luaL_checkstring(L, 2);
+	lString32 pattern		= lString32(l_pattern);
```

Comment:
Strange/incosistent spacing (tabs?).
(It's bad in other places, but let's make the new code tab less, at least no tab in the middle :)